File: app/src/main/java/com/an/github/utils/ShareUtils.java
package com.an.github.utils;

import android.app.Activity;
import android.support.v4.app.ShareCompat;

public class ShareUtils {

    public static void shareUrl(Activity activity,
                                String url) {
        ShareCompat.IntentBuilder
                .from(activity)
                .setType("text/plain")
                .setChooserTitle("Share URL")
                .setText(url)
                .startChooser();
    }
}

File: app/src/main/java/com/an/github/utils/ShareUtils.kt
package com.an.github.utils

import android.app.Activity
import android.support.v4.app.ShareCompat

object ShareUtils {

    fun shareUrl(activity: Activity,
                 url: String) {
        ShareCompat.IntentBuilder
                .from(activity)
                .setType("text/plain")
                .setChooserTitle("Share URL")
                .setText(url)
                .startChooser()
    }
}
